src/unpack/unpack_0_127.ts

```diff
@@ -30,12 +30,11 @@ export default function (incoming: Buffer, isoJSON: Types.KeyValueStringT, confi
     const bitmap = T.getHex(slice).split('').map(Number);
 
     let thisBuff = incoming.slice(bitmapEnd, incoming.byteLength);
-    for (let i = 1; i < bitmap.length; i++) {
-      if (bitmap[i] === 1) {
+    for (let field = 2; field <= bitmap.length; field++) {
+      if (bitmap[field - 1] === 1) {
         // format defined
-        const field = i + 1;
         const this_format = this.formats[field] || formats[field];
-        if (!config.skip127 && field === 127) {
+        if (!config.custom127Encoding && field === 127) {
```

Comment:
127 should be a const With a meaningful name.

Reply:
Would the meaning would be derived from the iso8583 spec?

Reply:
127 is already meaningful with the spec 🤷

Reply:
Not all people that work in here will know the spec.  It's about giving the next engineers a leg up.